apps/portal/src/content/docs/components/accordion.mdx

```diff
@@ -123,65 +167,174 @@ allows the display of `multiple` items or a `single` item. The accordion can be
 By default, the accordion is **uncontrolled** and will manage its own internal state. When the `value` and `onValueChange`
 props are supplied, the accordion becomes **controlled** and state is handled externally.
 
-```typescript jsx
-<Accordion.Root
-  type="single"                // single|multiple
-  collapsible                  // [OPTIONAL] allow all panels to be collapsed
-                               //            when type is "single"
-  defaultValue="ItemValue"     // [OPTIONAL] value of item to expand initially
-  disabled                     // [OPTIONAL] disable the accordion
-                               //            expanded when type is "single"
-  onValueChange={handleChange} // [OPTIONAL] handler triggered when value changes
-                               //            when externally controlled
-  value={currentValue}         // [OPTIONAL] value of item currently selected
-                               //            when externally controlled
->
-  {/* <Accordion.Item>'s as children */}
-</Accordion.Root>
-```
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "type",
+      description:
+        "Controls whether accordion allows display of multiple items or a single item",
+      defaultValue: "single",
+      required: true,
+      value: "'single' | 'multiple'",
+    },
+    {
+      name: "collapsible",
+      description: "Allow all panels to be collapsed when type is 'single'",
+      defaultValue: "false",
+      required: false,
+      value: "boolean",
+    },
+    {
+      name: "defaultValue",
+      description: "Value of item(s) to expand initially in uncontrolled mode",
+      required: false,
+      value: "string | string[]",
+    },
+    {
+      name: "disabled",
+      description: "Disable the accordion",
+      defaultValue: "false",
+      required: false,
+      value: "boolean",
+    },
+    {
+      name: "onValueChange",
+      description: "Handler triggered when value changes in controlled mode",
+      required: false,
+      value: "function",
```

Comment:
Can we swap `function` for the shape of the function to pass?

Reply:
Fixed